ppgan/apps/animegan_predictor.py

```diff
@@ -0,0 +1,73 @@
+import os
```

Comment:
Maybe also can add copyright, and the author zhen8838  after copyright. same as below